doc/architecture/decisions/0002-centralised-auth-routes.md

```diff
@@ -9,18 +9,18 @@ Under Review
 ## Context
 
 - Authentication and sign out are functionalities that must be maintained by all microservices, however during some users journeys they will traverse between our hub and submission services but this should be transparent to their experience.
-
-- The entry and exit points for the user sign/out in both microservices should be the same regardless whether the user executes it from the hub or submission service
+The entry and exit points for the user sign/out in both microservices should be the same regardless whether the user executes it from the hub or submission service
```

Comment:
remove the `-`, it makes this a bullet point

and add back the new line since it's a different paragraph